### PR TITLE
[17.0][MIG] hr_leave_custom_hour_interval: Migration to 17.0

### PR DESCRIPTION
**Pull request by xaviedoanhduy**
The failing test not related to this module

### CODE REVIEW

**Review by juanjosesegui-tecnativa — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@victoralmau can you check the failing test?

**Comment by juanjosesegui-tecnativa** [comment]
TT50071

**Comment by victoralmau** [comment]
To solve `hr_holidays_public` test it is only necessary to rebase 17.0, the error was solved at https://github.com/OCA/hr-holidays/commit/5770e4bc7326cd8431092ac6e2dad9ddcef6d5fa.

**Comment by pedrobaeza** [comment]
@xaviedoanhduy can you rebase?

**Comment by xaviedoanhduy** [comment]
Hello @pedrobaeza and @victoralmau, PR has been rebased, congratulations the test case failed has actually been solved. Thanks for your all support.

File: hr_leave_custom_hour_interval/readme/CONTRIBUTORS.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
-- Creu Blanca  
+- Creu Blanca
   - Olga Marco
   - Enric Tobella
+  - Duy Do Anh <duyda@trobz.com>
```
Comment (pedrobaeza) [inline hr_leave_custom_hour_interval/readme/CONTRIBUTORS.md]: Incorrect indentation. You don't belong to Creu Blanca.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**